pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -358,13 +359,28 @@ def installer_satellite(request):
     else:
         sat = lru_sat_ready_rhel(getattr(request, 'param', None))
     sat.setup_firewall()
-    # # Register for RHEL8 repos, get Ohsnap repofile, and enable and download satellite
+
+    # register to cdn (also enables rhel repos from cdn)
     sat.register_to_cdn(enable_proxy=True)
-    sat.download_repofile(
-        product='satellite',
-        release=settings.server.version.release,
-        snap=settings.server.version.snap,
-    )
+
+    # setup source repositories
+    if settings.server.version.source == "ga":
```

Comment:
`"ga"` ?!??
Wouldn't be `"cdn"` better ?

Reply:
This is a remainder of historical state when there were two versions available on cdn - ga and beta. Beta is no more used. However it still seems logical to me - we have internal source or generally available source. In case of renaming I would suggest different naming`internal`/`public`.